Matplotlib created a temporary config/cache directory at /tmp/matplotlib-t5djwkke because the default path (/home/fmriprep/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
You are using fMRIPrep-21.0.1, and a newer version of fMRIPrep is available: 21.0.2.
Please check out our documentation about how and when to upgrade:
https://fmriprep.readthedocs.io/en/latest/faq.html#upgrading
Traceback (most recent call last):
  File "/opt/conda/bin/fmriprep", line 8, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.8/site-packages/fmriprep/cli/run.py", line 38, in main
    parse_args()
  File "/opt/conda/lib/python3.8/site-packages/fmriprep/cli/parser.py", line 671, in parse_args
    opts = parser.parse_args(args, namespace)
  File "/opt/conda/lib/python3.8/argparse.py", line 1768, in parse_args
    args, argv = self.parse_known_args(args, namespace)
  File "/opt/conda/lib/python3.8/argparse.py", line 1800, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)
  File "/opt/conda/lib/python3.8/argparse.py", line 2006, in _parse_known_args
    start_index = consume_optional(start_index)
  File "/opt/conda/lib/python3.8/argparse.py", line 1946, in consume_optional
    take_action(action, args, option_string)
  File "/opt/conda/lib/python3.8/argparse.py", line 1874, in take_action
    action(self, namespace, argument_values, option_string)
  File "/opt/conda/lib/python3.8/site-packages/niworkflows/utils/spaces.py", line 708, in __call__
    for sp in Reference.from_string(val):
  File "/opt/conda/lib/python3.8/site-packages/niworkflows/utils/spaces.py", line 336, in from_string
    return [cls(_args[0], s) for s in allspecs]
  File "/opt/conda/lib/python3.8/site-packages/niworkflows/utils/spaces.py", line 336, in <listcomp>
    return [cls(_args[0], s) for s in allspecs]
  File "<attrs generated init niworkflows.utils.spaces.Reference>", line 11, in __init__
  File "/opt/conda/lib/python3.8/site-packages/niworkflows/utils/spaces.py", line 246, in _check_name
    raise ValueError(
ValueError: space identifier "MNI152NLin2009cAsym" is invalid.
Valid identifiers are: T1w, T2w, anat, fsnative, func, run, sbref, session, individual, dwi
